themes/website/static/js/osdetect.js

```diff
@@ -82,23 +82,11 @@ if (window.navigator.userAgent.indexOf("Android") != -1) {
 	OSName = "Android";
 
 	clients = [{
-		title: "Antox F-Droid",
-		name: "antox",
+		title: "aTox APK",
+		name: "atox",
 		icon: "external-link",
 		desc: true,
-		dlLink: "#fdroid",
-	}, {
-		title: "Antox Google Play",
-		name: "antox",
-		icon: "external-link",
-		desc: true,
-		dlLink: "https://play.google.com/store/apps/details?id=chat.tox.antox",
-	}, {
-		title: "Antox APK",
-		name: "antox",
-		icon: "download",
-		desc: true,
-		dlLink: "https://pkg.tox.chat/fdroid/repo/antox.apk",
+		dlLink: "https://github.com/evilcorpltd/aTox/releases/download/v0.5.1/atox.apk",
```

Comment:
do you really mean to version lock it to v0.5.1? Seems like it'll go out of date both quickly and often

Reply:
Well, you can't use `latest` for direct file downloads, e.g. `https://github.com/evilcorpltd/aTox/releases/download/latest/atox.apk` is a HTTP 404.

What do you suggest to do instead?

Reply:
Alright, let's merge this as it is and change the URL later when you reply with what you had on your mind. Doesn't make much sense to stall the PR for this.